src/auto/charset_converter.rs

```diff
@@ -0,0 +1,155 @@
+// This file was generated by gir (5a68ad0) from gir-files (469db10)
+// DO NOT EDIT
+
+use Error;
+use Initable;
+use ffi;
+use glib;
+use glib::StaticType;
+use glib::Value;
+use glib::object::Downcast;
+use glib::object::IsA;
+use glib::signal::SignalHandlerId;
+use glib::signal::connect;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::boxed::Box as Box_;
+use std::mem;
+use std::mem::transmute;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct CharsetConverter(Object<ffi::GCharsetConverter, ffi::GCharsetConverterClass>): Initable;
```

Comment:
This is useless without the `GConverter` interface (which has the actual API to make use of this)